Makefile

```diff
@@ -46,6 +47,9 @@ ARCH := $(shell uname -m)
 
 COMMIT_HASH := $(shell ./misc/hash.sh)
 
+PYVERSION := $(shell echo ${COMMIT_HASH} | sed 's/^v//;s/-/+/;s/-/./g')
```

Comment:
Specifically it puts the delta information that appears when the commit is not tagged in the local version bit which does not have any specific meaning but can distinguish archives with the same version.


https://www.python.org/dev/peps/pep-0440/#local-version-identifiers